### PR TITLE
feat:added blog for meshmap

### LINKED ISSUE #4520: [Blog]Guide to Publish designs on Meshery Playground for newcomers
Contributors and community members are encouraged to post on https://layer5.io/blog. A new blog post fitting the following description is requested to be written by a community member:

**Desired Title:**  Newcomer's Guide to Meshmap
**Topic:**  Meshmap
**Core points to convey in the post:** 
1. Introduction to Meshmap & Why Meshmap
2. How to get playground access
3. Meshmap tutorial and How to publish designs

---
<h3>Contributor Resources</h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by Ghat0tkach**
**Description**

This PR fixes #4520 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit f10913fe323aade4a8bd4340b42da2bf23cfe443 at: https://64afe46fa7039811efbfd09c--layer5.netlify.app

File: src/collections/blog/2023/2023-7-13-meshmap-guide-for-newcomers/index.mdx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import meshmap5 from "./meshmap5.webp";
+
+<BlogWrapper>
+<h2>What is Meshmap?</h2>
+<p> MeshMap specifically provides a way to visualize and understand how different services interact within a service mesh.It creates a graphical representation of the connections between different services and how data flows between them.</p>
+
+<h2>How to access Meshmap?</h2>
+<p>Visit the <a href="https://play.meshery.io/">Playground</a> and enter your credentials for the early access. Meshmap is an integrated part of meshery-cloud. Access to the Playground will be granted to you through your email at the soonest opportunity.</p>
+
+<h2>Why Meshmap??</h2>
```
Comment (theBeginner86) [inline src/collections/blog/2023/2023-7-13-meshmap-guide-for-newcomers/index.mdx:33]:
```suggestion
<h2>Why Meshmap?</h2> 
```

Comment (leecalcote) [reply src/collections/blog/2023/2023-7-13-meshmap-guide-for-newcomers/index.mdx:33]: Capitalize both “m”s

Comment (Ghat0tkach) [reply src/collections/blog/2023/2023-7-13-meshmap-guide-for-newcomers/index.mdx:33]: Thank you @theBeginner86

**Comment by l5io** [comment]
🚀 Preview for commit ae237b1f3fb873d9559f6de694082a3c6cea2316 at: https://64afef83d035c316d7606ef3--layer5.netlify.app

**Review by hirentimbadiya — COMMENTED** [review]
We can also specify an 'alt' tag in the 'img' attribute to provide alternative text for the images. Other than that, everything looks good to merge.

**Comment by Ghat0tkach** [comment]
> We can also specify an 'alt' tag in the 'img' attribute to provide alternative text for the images. Other than that, everything looks good to merge.

Thank you for the feedback  , added the alt tags as well!

**Comment by l5io** [comment]
🚀 Preview for commit fca2a8ff5239ef68cd920ce072ebfc2029fe4290 at: https://64b27f049a72cb7a84efdf80--layer5.netlify.app

**Comment by vishalvivekm** [comment]
@Ghat0tkach 

Let's discuss it on the websites call.
Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima), if you would :)

**Comment by l5io** [comment]
🚀 Preview for commit e4d0a27d42db08424a4a32304a2fcba53abeab22 at: https://64b520cd8d78930f097ffe5b--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit f2f7924559e9a441534327f8d1264813ec75c54f at: https://64bfe0e4679db90b0e277ca3--layer5.netlify.app

**Review by Chadha93 — CHANGES_REQUESTED** [review]
@Ghat0tkach, You can add links to meshery docs for publishing designs, and replace the video with "how to publish design".  
1. Docs - https://docs.meshery.io/extensions/publishing-a-design.
2. Video - https://www.youtube.com/watch?v=UCKS4eSB7AY

**Comment by Ghat0tkach** [comment]
@Chadha93 Thanks again , I will also need to add the design and community member profile

**Comment by l5io** [comment]
🚀 Preview for commit 996852bc2afb9e81b493d697c759c8c1564e7863 at: https://64c558bece1ad713497f7df4--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit be908721b1d82ddc3b841f0e3ea3741c56b523af at: https://64c55fbc61f7c9126830c8d8--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 5e4a5968b27f567a29942c96847a35e1f9363345 at: https://64c5642a4de7b11617518e9e--layer5.netlify.app

**Comment by Ghat0tkach** [comment]
Update : There is some issue in responsiveness since my member profile get added automatically .
Will fix this soon

**Comment by l5io** [comment]
🚀 Preview for commit 5416962d6c12fb64b473cf5d07e09f6f522ba693 at: https://64c8e46752a8f46f13b2ee95--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit fa45edb6580c6bec8eef8d60cde8ae83bb1d00db at: https://64d65dc554f3687d19ce58b9--layer5.netlify.app

**Comment by abhijeetgauravm** [comment]
> 🚀 Preview for commit [fa45edb](https://github.com/layer5io/layer5/commit/fa45edb6580c6bec8eef8d60cde8ae83bb1d00db) at: https://64d65dc554f3687d19ce58b9--layer5.netlify.app

image
last linking sentence @Ghat0tkach ok?

**Comment by Ghat0tkach** [comment]
> > 🚀 Preview for commit [fa45edb](https://github.com/layer5io/layer5/commit/fa45edb6580c6bec8eef8d60cde8ae83bb1d00db) at: https://64d65dc554f3687d19ce58b9--layer5.netlify.app
> 
> image last linking sentence @Ghat0tkach ok?

Thank you @abhijeetgauravm !🙌

**Comment by l5io** [comment]
🚀 Preview for commit 31f614e854d15ab2b0bc0d27a8630e082a1966fb at: https://64d806560c870d3e79251df3--layer5.netlify.app

**Comment by abhijeetgauravm** [comment]
> 🚀 Preview for commit [31f614e](https://github.com/layer5io/layer5/commit/31f614e854d15ab2b0bc0d27a8630e082a1966fb) at: https://64d806560c870d3e79251df3--layer5.netlify.app

image
hey @Ghat0tkach won't it look much appealing when  sentences between both images would be at top of the second image or somewhere else?

**Comment by Chadha93** [comment]
> > 🚀 Preview for commit [31f614e](https://github.com/layer5io/layer5/commit/31f614e854d15ab2b0bc0d27a8630e082a1966fb) at: [64d806560c870d3e79251df3--layer5.netlify.app](https://64d806560c870d3e79251df3--layer5.netlify.app)
> 
> image hey @Ghat0tkach won't it look much appealing when sentences between both images would be at top of the second image or somewhere else?

@Ghat0tkach, thoughts?

**Comment by Ghat0tkach** [comment]
> Member



> > > 🚀 Preview for commit [31f614e](https://github.com/layer5io/layer5/commit/31f614e854d15ab2b0bc0d27a8630e082a1966fb) at: [64d806560c870d3e79251df3--layer5.netlify.app](https://64d806560c870d3e79251df3--layer5.netlify.app)
> > 
> > 
> > image hey @Ghat0tkach won't it look much appealing when sentences between both images would be at top of the second image or somewhere else?
> 
> @Ghat0tkach, thoughts?

Yup , seems good !

**Comment by l5io** [comment]
🚀 Preview for commit 03cd8388504360b4760d7e4392eb4e3c35afb1af at: https://6510900d6686c813419ab732--layer5.netlify.app

**Comment by Ghat0tkach** [comment]
please have a review , gents

**Comment by leecalcote** [comment]
@Ghat0tkach, let's hold off on this until we officially launch MeshMap.

**Comment by iArchitSharma** [comment]
@vishalvivekm is it ready to merge

**Pull request closed without merging**